integration-test/src/test/java/com/datastrato/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -140,6 +181,62 @@ private static Network createDockerNetwork() {
             .build());
   }
 
+  public static boolean ipRangesOverlap(String cidr1, String cidr2) throws Exception {
+    long[] net1 = cidrToRange(cidr1);
```

Comment:
What the meaning of `cidr`?

Reply:
CIDR is https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing

Reply:
OK, can you include the link in the Java doc, others may also be confused by it.

Reply:
I added the comment about CIDR in the codes.
Another, This is only Integration Test, I think not need write in the Java doc.

Reply:
> I added the comment about CIDR in the codes.

That's exactly what I mean here.